apps/salga-mobile-api/salga-mobile-api-web.yml

```diff
@@ -0,0 +1,66 @@
+---
+- hosts:
+    - salga-mobile-api
+  become: yes
+  roles:
+    - dokku_bot.ansible_dokku
+
+  vars:
+    instance_name: "salga-mobile-api-{{ env_name }}"
+    app_name: "{{ instance_name }}-web"
```

Comment:
you can leave it like this for now, but most apps don't have the -web suffix, it's just that I didn't understand Procfiles when I initially set up gazettes